extensions/roblox-games/package.json

```
@@ -0,0 +1,90 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "roblox-games",
+  "title": "Roblox",
+  "description": "Searches for & launches Roblox places",
+  "icon": "command-icon.png",
+  "author": "TaylorRobinson",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Launch Roblox",
```

Judging by the screencast it seems like the command is more about games and searching for it. Should it be called something like
```suggestion
      "title": "Search Games",
```
?